testing/infra/terraform/modules/ec_deployment/deployment.tf

```diff
@@ -88,13 +107,36 @@ resource "local_file" "enable_expvar" {
   filename = "${path.module}/scripts/enable_expvar.sh"
 }
 
+resource "local_file" "secret_token" {
+  count = var.integrations_server ? 1 : 0
+  content = templatefile("${path.module}/scripts/secret_token.tftpl", {
```

Comment:
I can't see secret_token.tfpl, did you forget to add it?

Reply:
oops... I did...